### DISCUSSION TITLE
Create a sell on Instagram Plugin

### DISCUSSION THREAD

**Discussion by lukeholder** [craftcms/commerce#2486] [Plugin ideas]
Plugin would support the Facebook Commerce Platform for businesses that want to integrate their Facebook and Instagram Shops with native checkout.

https://developers.facebook.com/docs/commerce-platform/setup/getting-started

**Comment by masiorama** [comment]
Hello, what does it mean? The idea behind this is to ease the feeding of craft commerce products catalog to facebook and instagram to sell the products directly on social network shops?
